sycl/source/detail/scheduler/commands.cpp

```diff
@@ -694,6 +694,18 @@ void AllocaSubBufCommand::emitInstrumentationData() {
 #endif
 }
 
+void *AllocaSubBufCommand::getMemAllocation() const {
```

Comment:
Please, add override or final keyword.

Reply:
AFAIK, virt-specifiers (like `override` and `final`) are not allowed outside a class definition.

Reply:
Sorry, I though it is inside class :(.